ApplyToBecomeInternal/ApplyToBecome.Data/Models/AcademyConversion/LegalRequirements.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace ApplyToBecome.Data.Models.AcademyConversion
+{
+	public class LegalRequirements
+	{
+		public bool IsComplete { get; set; }
+		public ThreeOptions? GoverningBodyApproved { get; set; }
+		public ThreeOptions? ConsultationDone { get; set; }
+		public ThreeOptions? DiocesanConsent { get; set; }
+		public ThreeOptions? FoundationConsent { get; set; }
+
+		public Status Status =>
```

Comment:
I think this can be difficult to read when you use nested conditional ternary operator, could we consider using a switch statement or expression instead please.